web-registry/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -44,12 +45,12 @@ const RegistryLayoutHeader: React.FC = () => {
 
   const graphqlClient =
     useApolloClient() as ApolloClient<NormalizedCacheObject>;
-
+  const { data: csrfData } = useQuery(getCsrfTokenQuery({}));
   const { data: partyByAddr } = useQuery(
     getPartyByAddrQuery({
       client: graphqlClient,
       addr: wallet?.address ?? '',
-      enabled: isConnected && !!graphqlClient,
+      enabled: isConnected && !!graphqlClient && !!csrfData,
     }),
```

Comment:
wondering if there's no deeper issue that would explain why we need to verify the csrfToken is not empty for this particular query, but fine to merge as is for now since it solves the resulting bug

Reply:
I agree it's weird that this query requires a CSRF token to work.